bin/ert-runner

```diff
@@ -15,6 +20,13 @@ function has_option {
   return 1
 }
 
+inside_emacs_24
+if [[ $? == 1 ]]; then
```

Comment:
Yeah, you would have thought that, wouldn't you. But AFAICT, bash "functions" do not actually return values. So, I can't get this to work, and think that it doesn't.